java/java.source.base/src/org/netbeans/api/java/source/ElementUtilities.java

```diff
@@ -521,6 +523,21 @@ public Iterable<? extends TypeElement> getGlobalTypes(ElementAcceptor acceptor)
         return membersList;
     }
 
+    private static final Object ORDINARY_SCOPE_TYPE;
+    private static final Logger LOG = Logger.getLogger(ElementUtilities.class.getName());
+
+    static {
+        Object ordinary = null;
+
+        try {
+            ordinary = Enum.valueOf((Class<Enum>) Class.forName("com.sun.tools.javac.api.JavacScope$ScopeType"), "ORDINARY");
```

Comment:
The ScopeType is currently `private` in `JavacScope`. I'll see if there's something that can be done about it, but it seems somewhat unlikely it could be done in JDK24. So it seems we may need to live with code like this. My mistake.